pkg/storage/system_intake.go

```diff
@@ -49,6 +49,10 @@ func (s *Store) CreateSystemIntake(ctx context.Context, intake *models.SystemInt
 	if intake.DecisionState == "" {
 		intake.DecisionState = models.SIDSNoDecision
 	}
+	if intake.GrbReviewType == "" {
+		intake.GrbReviewType = models.SystemIntakeGRBReviewTypeStandard
+	}
+
 	const createIntakeSQL = `
```

Comment:
This works fine, up to you if you want to follow up on this suggestion. Since the SQL is setting a default value, you can just leave out setting the GRBReviewType, but still return the value. There isn't a way to create an intake and specify the type now, so you can rely on the SQL to update it.

Reply:
good enough for me! Thanks Tom.